habitat/tasks/rearrange/rearrange_sensors.py

```diff
@@ -559,6 +587,21 @@ def update_metric(self, *args, episode, task, observations, **kwargs):
         self._metric = self._accum_force
 
 
+@registry.register_measure
+class NumStepsMeasure(Measure):
+    cls_uuid: str = "num_steps"
+
+    @staticmethod
+    def _get_uuid(*args, **kwargs):
+        return NumStepsMeasure.cls_uuid
+
+    def reset_metric(self, *args, episode, task, observations, **kwargs):
+        self._metric = 0
+
+    def update_metric(self, *args, episode, task, observations, **kwargs):
+        self._metric += 1
```

Comment:
Can we always assume the measure is updated once per step? @mathfac is this is general design assumption? If not we may want to cache num_steps in the task and query it here instead.

Reply:
Yes, I think this can be assumed?